src/App.js

```diff
@@ -139,6 +143,27 @@ class App extends Component {
   };
 
   render() {
+
+    if ( 
+      this.props.availableOrganisations.length === 0 && 
+      this.props.isFetchingOrganisations === false &&
+      this.props.timesFetchedOrganisations > 0
+    ) {
+      const norolesMessage = this.props.intl.formatMessage({ id: "authorization.no_roles_message", defaultMessage: "Dear user, \nYou seem not to be in any organisations that can acces the management pages. \nTherefore you are redirected to the mainpage" });
+      alert(norolesMessage);
+      // should redirect to demo.lizard.net on prod
+      window.location = "/";
```

Comment:
Should it do something different on prod? I'm not convinced. Anyway either it has to do that or the comment can be removed.

Reply:
How can we test this on dev?
changed it to 
// should redirect to <customer_url>.lizard.net/management/ on prod

Reply:
changed comment.
difficult to test on dev